src/mmw/js/src/draw/views.js

```diff
@@ -172,12 +172,18 @@ var PlaceMarkerView = Marionette.ItemView.extend({
 });
 
 function changeOutlineLayer(endpoint, tableId, model) {
-    var map = App.getLeafletMap();
+    var map = App.getLeafletMap(),
+        ofg = model.get('outlineFeatureGroup');
 
-    if (endpoint && tableId != undefined) {
+    // If the outlines are already present, there is nothing more to do.
+    if (ofg.getLayers().length !== 0) {
```

Comment:
I do not have a firm understanding of the intended workflow, some possible variations that I can think of would preclude that as problem.  The other alternative is to simply clear the `outlineFeatureGroup` and re-add.  That actually probably makes more sense.